### ISSUE TITLE
any method to simplify the labeling process?

### ISSUE THREAD

**Issue by lihuibng** [OpenMatch/NeuScraper#4]
"The content extraction labels of ClueWeb22 were generated from the
production system of a commercial search engine.
The labels are not available for general web scraping tools, because they are annotated with more
expensive signals of page rendering and visualization. " I wonder if there is any method to do the labeling process by myself rather than using a commercial search engine.

**Comment by Zhipeng-Xu** [comment]
Hi @lihuibng 

I think the labeling process is very difficult and well beyond the scope of this project. Perhaps crowdsourcing is an option, but its cost is unacceptable. 
ClueWeb22's paper provides detailed information on their data annotation that may be helpful.
https://arxiv.org/abs/2211.15848

Feel free to ask again if you have more questions.

**Comment by Pab1x** [comment]
Hi @lihuibng

I think the labeling process is very difficult and far beyond the scope of this project. Perhaps crowdsourcing is an option, but its cost is unacceptable.
ClueWeb22's paper provides detailed information on their data annotation that may be helpful.
https://arxiv.org/abs/2211.15848

Feel free to ask again if you have more questions.

**Comment by lihuibng** [comment]
@Zhipeng-Xu  two more questions, 
1. Could you give some advice about extracting the Chinese website using this tools?
2. If I want to do a fresh training, what steps need I do ?

**Comment by Pab1x** [comment]
@lihuibng 

A1: neuscraper-v1-clueweb focuses on English corpus curation. It's probably feasible in Chinese. For NeuScraper, the recommended way to use it is to process CommonCrawl and use the language classifiers to pick the text in the language you want to use
A2: We have described the training process in the README, which should help you.

**Comment by lihuibng** [comment]
@Zhipeng-Xu I have another question about the document quality mentioned in the paper https://arxiv.org/abs/2211.15848, how to measure the quality of cleaned document or raw web page ? Does tf-idf work？

**Comment by Pab1x** [comment]
@lihuibng 
I think tf-idf struggles to handle this problem.

**Comment by lihuibng** [comment]
@Zhipeng-Xu I read the paper(https://www.andrew.cmu.edu/course/11-667/lectures/W2L1_pretraining_data.pdf), It uses the RoBERTa pretrain model's score based on the MNLI-m/mm benchmark to compare the quality of different datasets. Is it solid to compare the quality of the datasets like this? 
![image](https://github.com/OpenMatch/NeuScraper/assets/1087071/3767cae0-3ce8-43bc-a8d9-5f1d3392b935)

And are there any Chinese-version benchmark replacements for MNLI-m/mm?

**Comment by Pab1x** [comment]
@lihuibng 
I think it's reliable, Clueweb22 did the same thing in his analysis

**Comment by Pab1x** [comment]
That problem seems to have been solved

**Comment by lihuibng** [comment]
@Zhipeng-Xu thank you!

**Comment by lihuibng** [comment]
@Zhipeng-Xu, if I am going to do the RoBERTa pretrain based on the Chinese dataset, which open source repository do you recommend for me, [fairseq] (https://github.com/facebookresearch/fairseq/tree/main) or [roberta_zh] (https://github.com/brightmart/roberta_zh)? Or any other repository?

**Comment by Pab1x** [comment]
@lihuibng fairseq is a more standardized option, but I don't know how well it supports Chinese. Maybe some framework that has been adapted specifically for Chinese would be better.
